models/articles-model.js

```diff
@@ -16,21 +16,36 @@ exports.selectArticles = (articleId) => {
     });
 };
 
-exports.selectAllArticles = () => {
-  return db
-    .query(
-      `
-    SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COALESCE(COUNT(comments.article_id), 0) AS comment_count
-    FROM articles
-    LEFT JOIN comments ON articles.article_id = comments.article_id
-    GROUP BY articles.article_id
-    ORDER BY articles.created_at DESC
-  `
-    )
-    .then((result) => {
-      const articles = result.rows;
-      return articles;
-    });
+exports.selectAllArticles = (sortBy, order) => {
```

Comment:
This is where you could use default parameters.

Reply:
<img width="306" alt="Screenshot 2024-08-27 at 14 46 45" src="https://github.com/user-attachments/assets/0df10ba8-170b-40fb-8ee2-7e16e2e5e225">